js/modules/k6/execution/execution.go

```diff
@@ -176,8 +176,12 @@ func (mi *ModuleInstance) newTestInfo() (*goja.Object, error) {
 			}
 		},
 		"options": func() interface{} {
+			vuState := mi.vu.State()
+			if vuState == nil {
+				common.Throw(rt, fmt.Errorf("getting test options in the init context is not supported"))
```

Comment:
nit: `common.NewInitContextError` ?

Reply:
Good catch - fixed it for the others as well [d357f49](https://github.com/grafana/k6/pull/3708/commits/d357f4927290340bf66eb79af512215f20839c8d)

There is one more error, but it also doesn't work in `setup` so ... I decided to leave it for now.